content/hi/training/_index.html

```diff
@@ -0,0 +1,154 @@
+---
+title: प्रशिक्षण
+bigheader: कुबेरनेट्स प्रशिक्षण और प्रमाणन
+abstract: प्रशिक्षण कार्यक्रम, प्रमाणपत्र, और भागीदार।
+layout: basic
+cid: training
+class: training
+---
+
+<section class="call-to-action">
+  <div class="main-section">
+    <div class="call-to-action" id="cta-certification">
+      <div class="cta-text">
+        <h2>अपना क्लाउड नेटिव करियर बनाएं</h2>
+        <p>कुबेरनेट्स क्लाउड नेटिव आंदोलन के मूल में है। लिनक्स फाउंडेशन और हमारे प्रशिक्षण भागीदारों से प्रशिक्षण और प्रमाणन आपको अपने करियर में निवेश करने, कुबेरनेट्स सीखने और अपनी क्लाउड देशी परियोजनाओं को सफल बनाने की सुविधा देता है।</p>
+      </div>
+      <div class="cta-img-container">
+        <div class="logo-certification cta-image" id="logo-kcnf">
+          <img src="/images/training/kubernetes-kcnf-white.svg" />
+        </div>
+        <div class="logo-certification cta-image" id="logo-kcsa">
+          <img src="/images/training/kubernetes-cksa-white.svg" />
+        </div>
+        <div class="logo-certification cta-image" id="logo-cka">
+          <img src="/images/training/kubernetes-cka-white.svg"/>
+        </div>
+        <div class="logo-certification cta-image" id="logo-ckad">
+          <img src="/images/training/kubernetes-ckad-white.svg"/>
+        </div>
+        <div class="logo-certification cta-image" id="logo-cks">
+          <img src="/images/training/kubernetes-cks-white.svg"/>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section>
+  <div class="main-section padded">
+    <center>
+      <h2>edX पर निःशुल्क पाठ्यक्रम लें</h2>
+    </center>
+  <div class="col-container">
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>कुबेरनेट्स का परिचय <br> &nbsp;</b>
+        </h5>
+        <p>कुबेरनेट्स सीखना चाहते हैं? कंटेनरीकृत अनुप्रयोगों के प्रबंधन के लिए इस शक्तिशाली प्रणाली पर गहन प्राइमर प्राप्त करें।</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-kubernetes" target="_blank" class="button">कोर्स पर जाएँ</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>क्लाउड इंफ्रास्ट्रक्चर टेक्नोलॉजीज का परिचय</b>
+        </h5>
+        <p>ओपन सोर्स में अग्रणी, लिनक्स फाउंडेशन से सीधे क्लाउड प्रौद्योगिकियों के निर्माण और प्रबंधन की बुनियादी बातें सीखें।</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-cloud-infrastructure-technologies" target="_blank" class="button">कोर्स पर जाएँ</a>
```

Comment:
If you want to:
```suggestion
        <a href="https://www.edx.org/course/introduction-to-cloud-infrastructure-technologies" target="_blank" class="button"  hreflang="en">कोर्स पर जाएँ</a>
```

Reply:
@sftim  I like your suggestion as it will help the search engine to serve the result to users searching in that language.